### PR TITLE
hive: Add hive.Command()

### PR DESCRIPTION
**Pull request by joamaki**
hive.Command() now implements the hive related cobra commands, so they don't need to be repeated per binary. See commits for more details.

```
$ go run ./daemon hive
Cells:

  Ⓜ️ agent (Cilium Agent):
      Ⓜ️ infra (Infrastructure):
          Ⓜ️ gops (Gops Agent):
              ⚙️ (gops.GopsConfig) {
                  GopsPort: (uint16) 9890
              }
...

$ go run ./operator hive
Cells:
...

$ go run ./daemon hive dot-graph | dot -Tx11

```

### CODE REVIEW

**Review by pippolo84 — APPROVED** [review]
LGTM :100:

File: daemon/cmd/daemon_main.go [hunk 1]
```diff
@@ -882,10 +882,6 @@ func initializeFlags() {
 	flags.Int(option.EndpointQueueSize, defaults.EndpointQueueSize, "Size of EventQueue per-endpoint")
 	option.BindEnv(Vp, option.EndpointQueueSize)
 
-	flags.Duration(option.EndpointGCInterval, 5*time.Minute, "Periodically monitor local endpoint health via link status on this interval and garbage collect them if they become unhealthy, set to 0 to disable")
-	flags.MarkHidden(option.EndpointGCInterval)
-	option.BindEnv(Vp, option.EndpointGCInterval)
```
Comment (tklauser) [inline daemon/cmd/daemon_main.go:890]: If possible, maybe split out the flag removal into a dedicated commit? I saw the note in the commit message about it, but it looked a bit odd/unrelated in the diff.

**Review by tklauser — APPROVED** [review]
